### PR TITLE
DT-1743: 🐛 Fix context not being passed to bulk email

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/api/oauthApi.js [hunk 1]
```diff
@@ -34,7 +34,14 @@ const oauthApiFactory = (client, { clientId, clientSecret, url }) => {
   const del = (context, path) => client.del(context, path).then((response) => response.body)
   const currentUser = (context) => get(context, '/api/user/me')
   const currentRoles = (context) => get(context, '/api/user/me/roles')
-  const getUserEmail = (context, { username }) => get(context, `/api/user/${username}/email`)
+  const getUserEmail = async (context, { username }) => {
+    try {
+      return await get(context, `/api/user/${username}/email`)
+    } catch (error) {
+      if (error?.status === 404) return {}
```
Comment (petergphillips) [inline backend/api/oauthApi.js:41]: also fix email not found causing page not found for user details

File: backend/api/oauthApi.test.js [hunk 2]
```diff
@@ -95,21 +95,34 @@ describe('oauthApi tests', () => {
...
+      const actual = await oauthApi.getUserEmail(context, { username: 'joe' })
       expect(actual).toEqual(emailDetails)
     })
     it('should call user email endpoint', () => {
+      oauthApi.getUserEmail(context, { username: 'joe' })
       expect(client.get).toBeCalledWith(context, '/api/user/joe/email')
     })
+    it('should cope with not found from endpoint', async () => {
+      const error = new Error('User not found')
+      // @ts-ignore
```
Comment (JamieDraperUK) [inline backend/api/oauthApi.test.js:115]: If you wanted to get rid of this ignore you may be able to do: `const error = {...new Error('User not found'), status: 404)`

Comment (petergphillips) [reply backend/api/oauthApi.test.js:115]: cunning :grin:

**Review by JamieDraperUK — APPROVED** [review]
LGTM

**Pull request merged**